build.gradle

```diff
@@ -2,16 +2,41 @@ buildscript {
     repositories {
         google()
         jcenter()
+        maven {
+            url = "https://plugins.gradle.org/m2/"
+        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
-        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.5.3'
+        classpath 'com.android.tools.build:gradle:4.0.1'
+        classpath 'de.marcphilipp.gradle:nexus-publish-plugin:0.4.0'
+        classpath 'io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.21.2'
     }
 }
 
+plugins {
+    id 'io.codearte.nexus-staging' version '0.21.2'
+}
+
+version '2.0.1-SNAPSHOT'
+ext {
+    compileSdkVersion = 28
+    minSdkVersion = 19
```

Comment:
Most of our other projects are 21+. Do we want to bump this?

Reply:
Yeah @sarahkoop mentioned this also. I'm down for bumping it. Then we can do a major version bump for `popup-bridge-android` on the next release.